### ISSUE TITLE
[support] add read only mode

### ISSUE THREAD

**Issue by jianboy** [microsoft/vscode-remote-repositories-github#25]
For most scenarios, we only need to browse the code from github on tablet. The virtual keyboard always pops up automatically.

so, add a config in user setting:

```
 "remoteHub.readOnly": true,
```

**Comment by eamodio** [comment]
@isidorn Would setting the virtual file system to read-only actually avoid this?

**Comment by jianboy** [comment]
**I don't mean to modify the readability of the file. But disabled the function of editing code.**

Similar function, for example, preview markdown file, **I mean to preview code.**

**Comment by jianboy** [comment]
The editing window should not be editable and should lose focus.

**Comment by isidorn** [comment]
@eamodio yes it would, if the file system is readonly only -> every editor is redonly -> if the editor is redonly the virtual keyboard does not pop up.

More details about this topic can be found in this issue https://github.com/microsoft/vscode/issues/120943#issuecomment-858715499
